ci/environment_fermi.yml

```diff
@@ -52,3 +53,4 @@ dependencies:
   - rich
   - joblib
   - fermipy
+  - llvmlite
```

Comment:
`llvmlite` is available on `conda-forge` too, no need to add `numba` channel. Anyway I am not sure we should add this, it's not our direct dependency, rather something required by `numba`. Am I wrong?